accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -314,7 +306,6 @@ Windows-specific:
 
 * **X509Chain**
     - `.ctor(IntPtr)`
-* **X509Certificate2UI**
```

Comment:
it's definied in `System.Windows.Extensions.dll` which is marked as Windows-specific

https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Windows.Extensions/ref/System.Windows.Extensions.cs#L121

Reply:
Good catch!